src/live-updates.ts

```diff
@@ -0,0 +1,50 @@
+export type Entity = Record<string, unknown>;
```

Comment:
It might be nice to move this out to its own folder in the future, I can imagine that when we start introducing the changes this file will get massive. Could be good to separate concerns earlier. For example, merge graphQl and rest function could possibly even be their own files